test/testapps/BasicTestApp/EventCasesComponent.cshtml

```diff
@@ -0,0 +1,34 @@
+﻿<h3>Bubbling standard event</h3>
```

Comment:
@rynowak I now realise this E2E test duplicates some of what you're adding in #718. Hopefully we can combine the two sets of tests. I'm happy to rename this file and its corresponding `EventTest.cs`.